MSAL/test/unit/native_auth/network/responses/validator/MSALNativeAuthTokenResponseValidatorTests.swift

```diff
@@ -171,6 +171,34 @@ final class MSALNativeAuthTokenResponseValidatorTest: MSALNativeAuthTestCase {
             MSALNativeAuthTokenResponseError(error: .invalidGrant, subError: nil, errorDescription: nil, errorCodes: errorCodes, errorURI: nil, innerErrors: nil, continuationToken: nil)
         }
     }
+    
+    func test_invalidGrantMFARequired_triggerStrongAuthRequiredResponse() {
+        let continuationTokenResponse = "ct"
```

Comment:
Just to keep it consistent, can you call this one `continuationToken` or `continuationTokenResponse`?

Reply:
It is already called "continuationTokenResponse". Do you mean the value?

Reply:
Yes, I meant the value "ct"

Reply:
I think it's fair to leave the value as it is. There is no difference